### ISSUE TITLE
lint mode errors result in exit code 0

### ISSUE THREAD

**Issue by e14nielson** [aws-quickstart/taskcat#279]
**Describe the bug**
taskcat always returns 0 when in lint mode

**To Reproduce**

with a lint validated template
1. taskcat -c ci/taskcat.yml -l 
2. echo $?
0

with a template that fails lint
1. taskcat -c ci/taskcat.yml -l 
2. echo $?
0

**Expected behavior**
$? should be non 0

**Screenshots**
[taskcat] :Reading Config form: ci/taskcat.yml
[taskcat] |Queing test => test-account
[taskcat] |Queing test => test-dashboard
[taskcat] |Queing test => test-dlq
[taskcat] |Queing test => test-ingestion
[taskcat] |Queing test => test-tokens
[taskcat] |Queing test => test-webhook
[INFO   ] :Zipping lambda function webhook
[INFO   ] :Lint passed for test test-account on template /Users/ericnielson/Source/plaid-lambda/deployment/templates/account_cft.yaml:
[INFO   ] :Lint passed for test test-dashboard on template /Users/ericnielson/Source/plaid-lambda/deployment/templates/dashboard_cft.yaml:
[ERROR  ] :Lint detected issues for test test-dlq on template /Users/ericnielson/Source/plaid-lambda/deployment/templates/dlq_cft.yaml:
[ERROR  ] :    line 29 [E3012] [Check resource properties values] (Property Resources/DLQAlarm/Properties/Period should be of type
                       Integer)
[INFO   ] :Lint passed for test test-ingestion on template /Users/ericnielson/Source/plaid-lambda/deployment/templates/ingestion_cft.yaml:
[INFO   ] :Lint passed for test test-tokens on template /Users/ericnielson/Source/plaid-lambda/deployment/templates/tokens_cft.yaml:
[INFO   ] :Lint passed for test test-webhook on template /Users/ericnielson/Source/plaid-lambda/deployment/templates/webhook_cft.yaml:
[INFO   ] :Linting completed
DEP---C02XV2L9JG5H:deployment ericnielson$ echo $?
0

**Comment by e14nielson** [comment]
looking to get an update around this. ide like to use this for a stage in gitlab ci pipeline that will fail if the linter fails.

thanks

**Comment by andrew-glenn** [comment]
This will be addressed in v0.9.

**Comment by jaymccon** [comment]
implemented for both `lint` and `test run` in latest release:

```
$ taskcat lint
...
[ERROR  ] :     line 907 [3012] [Check resource properties values] Property Resources/GitPullLambda/Properties/Timeout should be of type
                                                         Integer
[ERROR  ] : Lint failed with errors

$ echo $status
1

$ taskcat test run
...
[ERROR  ] :     line 899 [3012] [Check resource properties values] Property Resources/GitPullLambda/Properties/MemorySize should be of type
                                                         Integer
[ERROR  ] : Lint failed with errors

$ echo $status
1
```
